server/paymentProviders/opencollective/manual.js

```diff
@@ -37,50 +36,15 @@ async function processOrder(order) {
     );
   }
 
-  // Pick the first that is set as a Number
-  const platformFeePercent = [
-    // Fixed in the Order (special tiers: BackYourStack, Pre-Paid)
-    order.data?.platformFeePercent,
-    // Fixed for Bank Transfers at collective level
-    order.collective.data?.bankTransfersPlatformFeePercent,
-    // Fixed for Bank Transfers at host level
-    // As of August 2020, this will be only set on a selection of Hosts (opensource 5%)
-    host.data?.bankTransfersPlatformFeePercent,
-    // Default for Collective (skipped for now)
-    // order.collective.platformFeePercent,
-    // Default to 0
-    0,
-  ].find(isNumber);
+  // In some tests, we don't have an order.paymentMethod set ...
+  if (!order.paymentMethod) {
+    order.paymentMethod = { service: 'opencollective', type: 'manual' };
+  }
 
-  // Pick the first that is set as a Number
-  const hostFeePercent = [
-    // Fixed in the Order (special tiers: BackYourStack, Pre-Paid)
-    order.data?.hostFeePercent,
-    // Fixed for Bank Transfers at collective level
-    // As of August 2020, this will be only set on a selection of Collective (some foundation collectives 5%)
-    order.collective.data?.bankTransfersHostFeePercent,
-    // Fixed for Bank Transfers at host level
-    // As of August 2020, this will be only set on a selection of Hosts (foundation 8%)
-    host.data?.bankTransfersHostFeePercent,
-    // Default for Collective
-    order.collective.hostFeePercent,
-    // Just in case, default on the platform
-    config.fees.default.hostPercent,
-  ].find(isNumber);
+  const platformFeeInHostCurrency = await getPlatformFee(order.totalAmount, order, host);
+  const hostFeeInHostCurrency = await getHostFee(order.totalAmount, order, host);
 
   const isFeesOnTop = order.data?.isFeesOnTop || false;
-  const feeOnTop = order.data?.platformFee || 0;
-
-  let platformFeeInHostCurrency;
-  if (isFeesOnTop) {
-    // If it's "Fees On Top", we're just using that
-    platformFeeInHostCurrency = feeOnTop;
-  } else {
-    //  Otherwise, use platformFeePercent
-    platformFeeInHostCurrency = -Math.round((platformFeePercent / 100) * order.totalAmount);
```

Comment:
@kewitz do you understand why it was negative here `-Math.round`?

Reply:
@znarf we store fees as a negative integer in our DB.